holocron/models/utils.py

```diff
@@ -8,7 +8,10 @@
 import torch.nn as nn
 from holocron.nn import BlurPool2d
 from typing import List, Optional, Any, Callable, Tuple
-from torchvision.models.utils import load_state_dict_from_url
+if int(torch.__version__.split('.')[1]) >= 10:
+    from torch.hub import load_state_dict_from_url
+else:
+    from torchvision.models.utils import load_state_dict_from_url
```

Comment:
Actually the function is available in PyTorch since 1.1.0 (https://github.com/pytorch/pytorch/blob/v1.1.0/torch/hub.py#L385), so no need for the conditional import

Reply:
Ok sorry I went a bit too fast, thanks for the fix